pandas/tests/indexes/test_common.py

```diff
@@ -391,3 +400,61 @@ def test_astype_preserves_name(self, index, dtype):
             assert result.names == index.names
         else:
             assert result.name == index.name
+
+
+@pytest.fixture(
+    params=[
+        tm.makeBoolIndex(10),
+        tm.makeFloatIndex(10),
+        tm.makeIntervalIndex(10),
+        tm.makeStringIndex(10),
+        tm.makeIntIndex(10),
+        tm.makeUIntIndex(10),
+        tm.makeRangeIndex(10),
+        tm.makeCategoricalIndex(10),
+        tm.makeMultiIndex(10),
+        tm.makeDateIndex(10),
```

Comment:
we *already* have the 'index' fixtures, pls use it.

Reply:
if you need a derivative one, then you can create it similary (e.g. use indices_dict). you *could* just add it in the pandas/conftest.py is fine.

Reply:
Sorry, didn't know that we keep all the general fixtures in `pandas/conftest.py`. I'll add another fixture there that adds missing values to the indices.